### PR TITLE
[stable/grafana] Bugfix secrets for sidecar role

### PR DESCRIPTION
**Pull request by fr-ser**
Give the secrets authorization also to the role and not only the clusterrole

#### Which issue this PR fixes
  - fixes: Add secrets resource to role (not only clusterrole)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by zanhsieh — APPROVED** [review]

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 3.7.0
+version: 3.7.1
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: Please bump to 3.7.2. Thank you.

Comment (fr-ser) [reply stable/grafana/Chart.yaml]: Updated

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**